### ISSUE TITLE
Calls to 'setActive' reports errors on mods with disabled plugins

### ISSUE THREAD

**Issue by mdavis199** [ModOrganizer2/modorganizer#1258]
This is a followup to #1219.   It seems that when ```self._organizer.modList().setActive(names, False)``` is called for mods that have inactive plugins, MO spews error messages to its log.  

In one call to setActive(), I have 619 errors messages similar to the following: 
```
[2020-10-08 15:07:32.590 W] Plugin not found: 1nivWICCloaks.esp
[2020-10-08 15:07:32.590 W] Plugin not found: 1nivWICCloaks_COS_CCOR_Patch.esp
...
```
Other than the messages, the call to setActive seems to work fine.    The version I'm using is the github master built a few minutes ago.

**Comment by Holt59** [comment]
Does this also happens if you disable the mods in the UI (select > right click > disable selected or something like that)?

**Comment by mdavis199** [comment]
I didn't notice before, but yes it does. It happens in the GUI as well.

**Comment by Holt59** [comment]
I think the problem is that you don't often disabled mods in bulk with the GUI, so these messages are not annoying in this case. We have to decide what to do with these, or add a way for plugins to disable them.

**Comment by mdavis199** [comment]
I should have been clearer.  It happens if I just click with the mouse on a single mod to deactivate it and that mod has disabled plugins.  I don't think this is a bulk thing

**Comment by Holt59** [comment]
> 
> 
> I should have been clearer. It happens if I just click with the mouse on a single mod to deactivate it and that mod has disabled plugins. I don't think this is a bulk thing

Yes, my point is that this gets only "noticeable" if you deactivate a bunch of plugins at once. But I'm not sure these messages should be there in the first place, even for single mods.

**Comment by Holt59** [comment]
Could you give an example of plugins for which you have these messages (mod states + plugin states when you disable)?

**Comment by mdavis199** [comment]
I'm guessing you are asking which game plugin/mod  not MO plugin.     Actually, the problem appears anytime a mod is being disabled, regardless of the state of any plugin.   Here are some results:

Starting point: No mods enabled.
1. enable Alternate Start - Live another Life
  - Now: Both the mod  and the plugin "Alternate Start - Live another Live.esp" is enabled
2. uncheck the mod: Alternate Start 
  - No mods are enabled
  - Gives an error message "Plugin not found: Alternate Start - Live Another Life.esp

So it looks like it happens whenever a mod (that contains a plugin) is deactivated.
If a mod has more that one plugin. say: "Lexy's LOTD Consistency Patches - ELE Merge", I get an error message for each plugin in the mod.

**Comment by Holt59** [comment]
Thanks for the feedback. I took the liberty changing the title of this issue and editing the original post because those are actually not related to `IModList::setActive` at all but rather to some "bugs" in one of the game plugin (the meta plugin for Bethesda games actually).
